setup.js

```diff
@@ -0,0 +1,7 @@
+/* eslint-disable no-unused-vars */
+import 'dotenv/config';
+import { client } from './src/utils/db.js';
+import { User } from './src/models/user.js';
+import { Token } from './src/models/token.js'
```

Comment:
There is a missing semicolon at the end of this import statement. While JavaScript can often handle this, it's a good practice to include semicolons to avoid potential issues.